src/style.css

```diff
@@ -1,3 +1,73 @@
+html {
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. For example, you can use 'font-family: Roboto, Arial, sans-serif;' to ensure that if 'Roboto' is not available, 'Arial' will be used, and if neither is available, the default sans-serif font will be used.